src/malloc/mallocng/aligned_alloc.c

```diff
@@ -1,7 +1,11 @@
+#define INTERNAL_MUSL
```

Comment:
Why are these defines needed?

Reply:
In the /include/stdlib.h header file the malloc functions are given the cheerp_asmjs attribute only if INTERNAL_MUSL is defined. Otherwise they don't compile due to incompatible attributes.
This define is also present in oldmalloc.

Reply:
I am not convinced this thing make sense. Is there ever a case in which malloc and friend should _not_ be tagged? Please discuss with yuri

Reply:
it's to make it work when compiling with the genericjs target but mallocing asmjs structs